cmd/snaptel/task.go

```diff
@@ -364,6 +364,22 @@ func createTaskUsingTaskManifest(ctx *cli.Context) error {
 	case ".json":
 		e = json.Unmarshal(file, &t)
 		if e != nil {
+			if jsonError, ok := e.(*json.SyntaxError); ok {
```

Comment:
Instead of duplicating whole block of code with long error messages, consider moving it to separate function, for example handleJSONSyntaxError(file, error) and use just like:
```go
e = json.Unmarshal(file, &t)
if e != nil {
   handleJSONSyntaxError(file, e)
}
```